server/src/main/java/org/opensearch/wlm/ResourceType.java

```diff
@@ -18,8 +18,10 @@
 
 /**
  * Enum to hold the resource type
+ * 
+ * @opensearch.api
  */
-@PublicApi(since = "2.x")
+@PublicApi(since = "2.17.0")
```

Comment:
@kaushalmahi12 as an author of original change, `2.x` is not a version but branch name, please annotate APIs with versions they are supposed to be released, thank you.